modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -17,6 +17,53 @@ See the xref:project-docs:sdk-full-installation.adoc[Full Installation] guide fo
 
 
 // tag::all[]
+== Version 3.4.4 (8 March 2023)
+This is a regular maintenance release.
+
+https://packages.couchbase.com/clients/java/3.4.4/Couchbase-Java-Client-3.4.4.zip[Download] |
+https://docs.couchbase.com/sdk-api/couchbase-java-client-3.4.4/index.html[API Reference] |
+http://docs.couchbase.com/sdk-api/couchbase-core-io-2.4.4/[Core API Reference]
+
+The supported and tested dependencies for this release are:
+
+* io.projectreactor:**reactor-core:3.5.0**
+* org.reactivestreams:**reactive-streams:1.0.4**
+
+Optional artifacts on top of this SDK version are tested for the following compatibilities:
+
+.Optional Artifact Version Compatibility
+[options="header"]
+|=======================
+| Artifact                  | Couchbase Version | Built Against              | API Stability
+| `tracing-opentelemetry`   | 1.2.4             | OpenTelemetry 1.19.0       | Committed
+| `tracing-opentracing`     | 1.2.4             | OpenTracing 0.33.0         | Committed
+| `metrics-opentelemetry`   | 0.4.4             | OpenTelemetry 1.19.0       | Volatile
+| `metrics-micrometer`      | 0.4.4             | Micrometer 1.10.0          | Volatile
+|=======================
+
+
+=== Improvements
+* https://issues.couchbase.com/browse/JCBC-2050[JCBC-2050]:
+Coordinates in Full-Text Search geographic queries can now be specified using a staged builder, so it's harder to accidentally swap the latitude and longitude components. Example usage:
+```
+Coordinate eiffelTower = Coordinate.lat(48.858093).lon(2.294694);
+```
+Or, if you prefer to specify longitude first:
+```
+Coordinate eiffelTower = Coordinate.lon(2.294694).lat(48.858093);
+```
```

Comment:
Even in a relnote, we'd generally prefer a

[source,foo]
----
code here
----

style call out, to pull in the copyable text page furniture,
but I'll put this through as-is for now.